setup.py

```diff
@@ -28,7 +28,7 @@
 BASE_DIR = os.path.join(os.path.expanduser("~"), ".indy")
 
 tests_require = ['attrs==19.1.0', 'pytest==3.3.1', 'pytest-xdist==1.22.1', 'pytest-forked==0.2',
-                 'python3-indy==1.13.0-dev-1420', 'pytest-asyncio==0.8.0']
+                 'python3-indy==1.15.0', 'pytest-asyncio==0.8.0']
```

Comment:
@jovfer, @askolesov, @Toktar, What is your opinion on this?  Do you anticipate any unforeseen issues here.

Reply:
I think it's ok if tests are passing

Reply:
They are.

Reply:
@WadeBarnes, @m00sey I really like this idea. But if we change the dependency here, I suggest changing them in `ci/uduntu.dockerfile` to preservation of consistency even if this file will be deprecated soon.
And it's a bit strange if indy-node and indy-plenum will depend on different libindy packages. I guess it's absolutely unimportant because of it's used just for a test environment, but still.